sdk/cdn/Azure.ResourceManager.Cdn/src/autorest.md

```diff
@@ -198,24 +210,6 @@ directive:
   - from: swagger-document
     where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/afdEndpoints/{endpointName}'].put.parameters[3]
     transform: $['x-ms-client-name'] = 'endpointInput'
-  - rename-operation:
```

Comment:
Considering that all of these request paths are related to get log information for Azure Front Door, I prefer to keep these renaming.

Reply:
Removing these does not change anything in public API. I added some operation name overriding to ensure we have comprehensive operation names - see L28-L33 in this file.
And since we are now writing operationId and request path in the operation summary, if possible, we should not change it for future swagger references.